src/jobtypes.py

```diff
@@ -68,11 +67,31 @@ def __init__(self, repo_name='mozilla-inbound'):
                 #  opt test web-platform-tests-4"
                 if job['build_system_type'] == 'buildbot':
                     testtype = job['ref_data_name'].split(' ')[-1]
+                    job_identifier = job['ref_data_name']
 
                 # taskcluster's testtype is a part of its 'job_type_name' like reftest-2
                 # for [TC] Linux64 reftest-2
                 else:
-                    testtype = job['job_type_name'].split(' ')[-1]
+                    # we don't want job of build kind in seta
+                    if job['ref_data_name'].startswith('desktop-test') or \
+                            job['ref_data_name'].startswith('android-test'):
+                        continue
```

Comment:
I think the logic here is backwards, we want all tests that start with desktop-test and android-test